frontend/src/components/forms/CreateLog.tsx

```diff
@@ -276,44 +267,44 @@ const CreateLog = () => {
                       options={EMPLOYEES}
                       isDisabled
                       defaultValue={{ label: employee, value: employee }} // needs to be the current user
-                      styles={{
-                        control: (provided, state) => ({
-                          ...provided,
-                          border: getBorderStyle(state, employeeError),
-
-                          borderRadius: "4px",
-                        }),
-                      }}
+                      styles={selectStyle}
                     />
                   </FormControl>
                 </Col>
                 <Col>
                   <Grid templateColumns="repeat(2, 1fr)" gap="8px">
                     <GridItem>
-                      <FormControl isRequired>
+                      <FormControl
+                        isRequired
+                        style={{
+                          borderColor: colors.gray[200],
+                        }}
+                      >
                         <FormLabel>Date</FormLabel>
                         <SingleDatepicker
                           name="date-input"
                           date={date}
                           onDateChange={handleDateChange}
-                          propsConfigs={{
-                            popoverCompProps: {
-                              popoverContentProps: {
-                                background: "white",
-                              },
-                            },
-                          }}
+                          propsConfigs={singleDatePickerStyle}
                         />
                       </FormControl>
                     </GridItem>
                     <GridItem>
-                      <FormControl isRequired isInvalid={timeError}>
+                      <FormControl
+                        isRequired
+                        isInvalid={timeError}
+                        style={{ color: colors.gray[500] }}
+                      >
                         <FormLabel>Time</FormLabel>
                         <Input
                           size="md"
                           type="time"
                           defaultValue={time}
                           onChange={handleTimeChange}
+                          style={{
```

Comment:
the inputs are time, date and note. It might make sense to create a file like `formInput.tsx` and add the styling there so we can reuse it for the other forms